ptvsd/wrapper.py

```diff
@@ -1864,6 +1886,32 @@ def version_str(v):
         }
         self.send_response(request, **sys_info)
 
+    # VS specific custom message handlers
+    @async_handler
+    def on_setDebuggerProperty(self, request, args):
+        jmc = int(args.get('JustMyCodeStepping', 0))
+        self.debug_options['JUST_MY_CODE'] = jmc > 0
+        self.send_response(request)
+
+    def _is_stdlib(self, filepath):
+        for name in IMPORTLIB_BOOTSTRAP:
+            if filepath.endswith(name):
+                return True
+        for prefix in STDLIB_PATH_PREFIXES:
```

Comment:
If `IDE_PROJECT_ROOTS` and `LIBRARY_ROOTS` are properly set (after I'm able to integrate my latest changes) this could be handled by pydevd directly).

-- I plan on providing a pull request on Monday to address it as well as the other things I've worked on now that the pull request to move pydevd to 1.2 was integrated.